### ISSUE TITLE
Find purpose of Iridium GPS column

### ISSUE THREAD

**Issue by Huh** [Huh/collar#16]
How is it different than GPS Latitude and why do we care.

**Comment by rgzn** [comment]
Can you give an example of this column and its data?

**Comment by Huh** [comment]
One example from Telonics


Acquisition Time | Acquisition Start Time | Iridium CEP Radius | Iridium Latitude | Iridium Longitude | GPS Fix Time | GPS Fix Attempt | GPS Latitude | GPS Longitude | GPS UTM Zone | GPS UTM Northing | GPS UTM Easting | GPS Horizontal Error | GPS Navigation Time | GPS Geofence Count | GPS Geofence List | Activity Count | Temperature | Satellite Uplink | Receive Time | Repetition Count | Low Voltage | Mortality | Iridium Command | Schedule Set | Predeployment Data | Error
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2018.07.18 18:01:18 | 2018.07.18 18:00:00 |   |   |   | 2018.07.18 18:01:18 | Succeeded | 47.509673 | -113.939043 | 12T | 5265992 | 278700 | 6.5 | 78 | 0 |   |   |   |   | 2018.07.20 15:01:42 | 1 |   |   |   | Primary (outside) | No |  
2018.07.18 21:30:41 | 2018.07.18 21:30:00 |   |   |   | 2018.07.18 21:30:41 | Succeeded | 47.508375 | -113.935556 | 12T | 5265838 | 278957 | 7.5 | 41 | 0 |   |   |   |   | 2018.07.20 15:01:42 | 1 |   |   |   | Primary (outside) | No |  


I do not know the purpose of the column.  This and other data examples can be found in /inst/extdata

You have to register to see a manual and I have not done that yet.

**Comment by Huh** [comment]
I did not see other examples of this column or anything specifically called Iridium GPS, so hasty issue creation may be causing a bit of confusion.

**Comment by rgzn** [comment]
TLDR: I think this is an unused "fix at time of iridium transmission"

Long answer:
In Iridium short burst data messages, location is one of the optional fields. Some iridium messaging units will choose to send a location with each message, which would be the location at time of iridium transmission. My guess is this was left in but unused, as GPS collars are going to be transmitting fixes recorded at specified times, and not a fix at time of transmission.

**Comment by Huh** [comment]
Great, thanks
